scripts/examples/python/xva.py

```diff
@@ -61,7 +61,7 @@ def save(self, fileobj):
         output = tarfile.TarFile(mode='w', fileobj=fileobj)
         tarinfo = tarfile.TarInfo("ova.xml")
         tarinfo.size = len(ova_txt)
-        output.addfile(tarinfo, StringIO.StringIO(ova_txt))
+        output.addfile(tarinfo, io.StringIO(ova_txt))
```

Comment:
Requesting a change and a test here, if the script is used (not obsolete):

From my experience with the Python3 changes for https://github.com/xenserver/status-report I am quite certain that using io.StringIO as the IO for Python3's `tarfile` implementation will not work.

In Python2, `str` and `bytes` are the same type, so it works, but AFAIK, Python3 `tarfile` won't be able to work with Pyton3's *Unicode* str for I/O, it still needs the I/O based on bytes. Therefore, `io.BytesIO()` would likely have to be used to arrive at actually working, correctly running code that can be covered by executing a test case in CI.

Reply:
I do not have information about xva.py being obsolete .i will  make above changes and test the code.

Reply:
After  Edwin Torok and Rob Hoes approval  deleted xva.py under /xen-api/scripts/examples/python/   commit 2a6aaaea534544cc7ebe5dcde4b78cc4bc94a106